src/shim/conf/config.toml

```diff
@@ -0,0 +1,10 @@
+log_level = "info" # "debug" "info" "warn" "error"
+
+[containerd]
+    # Pre-installed OCI bundle for agent enclave container
+    agent_container_instance = "/opt/enclave-cc/agent-instance/"
+    # The root dir of agent enclave container running state
+    agent_container_root_dir = "/run/containerd/agentenclave"
```

Comment:
Consistency + legibility: If this is a path we control (?), please add a dash in this to make it clearer: `agent-enclave`.